### DISCUSSION TITLE
Have you a mod conflict list?

### DISCUSSION THREAD

**Discussion by fromery** [AnOpenSauceDev/Methane-mod#103] [Q&A] [answered]
Methane 3.8.1
MC 1.20.4
Singleplayer

I've tried to load (existing one) or create a new world with Methane but nothing. All stops at 0%.
terminal doesn't show anything like it freezes.

I have many mods (~250) and have no clue where to find a conflict.

So have you some mod conflict list?

**Comment by AnOpenSauceDev** [comment, marked as answer]
Almost 100% sure it's Starlight. Nothing else really causes a crash with Methane, and if it does it's really easy to figure out what caused it.

**Reply by fromery** [reply]
Yep! Thanks a lot!
